resultsdbupdater/utils.py

```diff
@@ -16,15 +19,37 @@
     format=log_format, level=CONFIG.get('resultsdb-updater.log_level'))
 
 
+def retry_session():
+    # This will give the total wait time in minutes:
+    # >>> sum([min((0.3 * (2 ** (i - 1))), 120) / 60 for i in range(24)])
+    # >>> 30.5575
+    # This works by the using the minimum time in seconds of the backoff time
+    # and the max back off time which defaults to 120 seconds. The backoff time
+    # increases after every failed attempt.
+    session = requests.Session()
+    retry = Retry(
```

Comment:
Ah, I think you need to add 'POST' to the `method_whitelist` argument to this `Retry` initializer.

See http://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html#urllib3.util.retry.Retry

Reply:
Good catch. I fixed it in the latest rebase.